charts/trusted-artifact-signer/templates/cosign-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         - name: FULCIO_URL
           value: "https://fulcio.$(OPENSHIFT_APPS_SUBDOMAIN)"
         - name: OIDC_ISSUER_URL
-          value: "https://keycloak-keycloak-system.$(OPENSHIFT_APPS_SUBDOMAIN)/auth/realms/$(OIDC_AUTHENTICATION_REALM)"
+          value: "https://oauth2.sigstore.dev/auth"
```

Comment:
Why change this to the PGI?

Reply:
@JasonPowr @cooktheryan and I were discussing in slack - by defaulting to this, the stack will work out of the box regardless of whether you configure an OIDC provider. We never thought of this until now, but it seems to be a good default option.